public/react-services/error-orchestrator/error-orchestrator.factory.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Wazuh app - Error Orchestrator factory
+ * Copyright (C) 2015-2021 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+
+import { ErrorOrchestrator, UIErrorSeverities, UIErrorSeverity } from '../../../common/constants';
+import { ErrorOrchestratorUI } from './error-orchestrator-ui';
+import { ErrorOrchestratorBusiness } from './error-orchestrator-business';
+import { ErrorOrchestratorCritical } from './error-orchestrator-critical';
+
+export const errorOrchestratorFactory = (severity: UIErrorSeverity): ErrorOrchestrator => {
+  switch (severity) {
+    case UIErrorSeverities.UI:
+      return new ErrorOrchestratorUI();
+    case UIErrorSeverities.BUSINESS:
+      return new ErrorOrchestratorBusiness();
+    case UIErrorSeverities.CRITICAL:
+      return new ErrorOrchestratorCritical();
+    default:
+      break;
+  }
```

Comment:
Why do we have multiple constructors of loggers depending on `severity`? Could this be a property/param and use a logger?

Reply:
The idea is that they are independent, that they can be used individually, and thus by extending `ErrorOrchestratorBase`, they can scale better.
But it is open to suggestions obviously 😃